src/Repository/Doctrine/NodeRepository.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Adshares\AdsOperator\Repository\Doctrine;
+
+use Adshares\AdsOperator\Document\Node;
+use Adshares\AdsOperator\Repository\NodeRepositoryInterface;
+use Doctrine\ODM\MongoDB\DocumentRepository;
+use Doctrine\ODM\MongoDB\MongoDBException;
+
+class NodeRepository extends DocumentRepository implements NodeRepositoryInterface
+{
+    public function availableSortingFields(): array
+    {
+        return [
+            'id',
+            'balance',
+        ];
+    }
+
+    public function findNodes(string $sort, string $order, int $limit, int $offset): array
+    {
+        $nodes = [];
+
+        try {
+            $cursor = $this
+                ->createQueryBuilder()
+                ->sort($sort, $order)
+                ->limit($limit)
+                ->skip($offset)
+                ->getQuery()
+                ->execute();
+
+            $data = $cursor->toArray();
+
+            foreach ($data as $id => $node) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "$id" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1481)